ibis/omniscidb/client.py

```diff
@@ -315,6 +315,27 @@ def load_data(self, df):
         stmt = ddl.LoadData(self._qualified_name, df)
         return self._execute(stmt)
 
+    def read_csv(self, path, header=True, quoted=True):
```

Comment:
hey @igalink I like the idea of adding `read_csv` into Ibis world! thanks for working on that.

maybe would be good to use as much as possible the same API of `pandas.read_csv` https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html

for your proposal use the same parameters/types used by pandas.

you don't need to implement all the parameters, but add `**kwargs` at the end of the function signature.

let me know if you have any question.

Reply:
also add some test for this method, please :)

Reply:
Since `pandas.read_csv` is already so strong, why not leave it to pandas and just implement the `create_table()` function so that if a `pandas.DataFrame` is passed instead of an expression, that it uploads/inserts that data after creating the table?

Reply:
I think maybe for large data it will be very slow ... 

also there is already a load_data method inside omniscidb https://github.com/ibis-project/ibis/blob/master/ibis/omniscidb/client.py#L302

Reply:
Now I'm starting to understand what the purpose of this is. Still have a question, though. Does the file/path have to reside on the server running Omnisci? Or can it reside locally?

Reply:
Hello ALL! Thanks for comments!
I'm sorry that have not learned code verification environment before creating a PR. All fixes and tests will be added soon. 
The goal is to load large data files without additional copying, so "COPY FROM" omnisci sql statement is used.
I think this function could be more generic to cover all supported data file formats but still it loads data from file without any intermediate data structures.

Reply:
thanks @igalink let us know if you have any question.